src/postings/skip.rs

```diff
@@ -6,6 +6,13 @@ use crate::query::Bm25Weight;
 use crate::schema::IndexRecordOption;
 use crate::{DocId, Score, TERMINATED};
 
+// doc num bits between 0..=32 are used for legacy, delta without -1 encoding
+// 33..=65 are used for current delta-1 encoding
```

Comment:
I suspect bit 7 would have been nicer. Of course it is not super natural since our values go from 0 to 32 included. (hence not fitting on 5 bits)

```
let delta_1 = (encoding_flag >> 7) & 1;
let num_bits = (encoding_flag & MASK);
```